README.md

```diff
@@ -1,10 +1,154 @@
+    - [Speeding up import](#speeding-up-import)
+    - [Importing changes](#importing-changes)
+  - [Configuration](#configuration)
+    - [Java version](#java-version)
+    - [Server version](#server-version)
+  - [Workplace Diagnostic](#workplace-diagnostic)
+  - [Run doctor](#run-doctor)
+  - [All Available Commands](#all-available-commands)
+  - [Show document symbols](#show-document-symbols)
+  - [Formatting on save](#formatting-on-save)
+  - [Status Bar](#status-bar)
+  - [Troubleshooting](#troubleshooting)
+
+
+
+### Requirements
+
+- Sublime text 3 or above
+- [LSP](https://github.com/tomv564/LSP) package: `Command Palette (Cmd + Shift + P) > Install package > LSP`
+- Java 8 or 11 provided by OpenJDK or Oracle. Eclipse OpenJ9 is not supported,
+    please make sure the JAVA_HOME environment variable points to a valid Java 8 or
+    11 installation.
+
+### Installing LSP-metals
+
+Once you have `LSP` installed, you can then install Metals via
+
+`Command Palette (Cmd + Shift + P) > Install package > LSP-metals`
+
+### Importing a build
+
+The first time you open Metals in a new workspace it prompts you to import the
+build. Click "Import build" to start the installation step.
+
+![Build Import](https://i.imgur.com/eUk30Zy.png)
+
+- "Not now" disables this prompt for 2 minutes.
+- "Don't show again" disables this prompt forever, use rm -rf .metals/ to
+    re-enable the prompt.
+- Run `lsp toggle server panel` in the command palette to watch the build import progress. You can optionally add key binding for this command.
+- Behind the scenes, Metals uses Bloop to import sbt builds, but you don't need
+    Bloop installed on your machine to run this step.
+
+Once the import step completes, compilation starts for your open *.scala files.
+
+Once the sources have compiled successfully, you can navigate the codebase with
+goto definition.
+
+#### Speeding up import
+
+The "Import build" step can take a long time, especially the first time you run
+it in a new build.  The exact time depends on the complexity of the build and if
+library dependencies need to be downloaded. For example, this step can take
+everything from 10 seconds in small cached builds up to 10-15 minutes in large
+uncached builds.
+
+Consult the Bloop [documentation](https://scalacenter.github.io/bloop/docs/what-is-bloop) to learn how to speed up build import.
+
+#### Importing changes
+
+When you change build.sbt or sources under project/, you will be prompted to
+re-import the build.
 
 ### Configuration
 
 Configure the metals language server by accessing `Preferences > Package Settings > LSP > Servers > LSP-metals`.
+
+![config](https://i.imgur.com/WFSJKV0.png)
+
+#### Java version
+The `LSP-metals` extension uses by default the `JAVA_HOME` environment variable
+(via [`environ`](https://docs.python.org/3/library/os.html#os.environ)), otherwise uses [which](https://docs.python.org/3/library/shutil.html#shutil.which) to locate the `java` executable.
```

Comment:
```suggestion
(via [`environ`](https://docs.python.org/3/library/os.html#os.environ)). Otherwise, it uses [which](https://docs.python.org/3/library/shutil.html#shutil.which) to locate the `java` executable.
```